### PR TITLE
Add test to resize vm with startstop

### PR DESCRIPTION
**Pull request by adityagesh**
Add Testcase to resize VM with StartStop
Rename TestSuite Name VmHotResize -> VmResize
Rename TestSuite Area vm_hot_resize -> vm_resize
Rename vm_hot_resize to vm_resize to accommodate
both tests in same file.

### CODE REVIEW

File: microsoft/testsuites/core/vm_resize.py [hunk 1]
```diff
@@ -59,12 +59,54 @@ def verify_vm_hot_resize(self, log: Logger, node: Node) -> None:
...
 
-    def _verify_vm_hot_resize(
-        self, node: Node, resize_action: ResizeAction = ResizeAction.IncreaseCoreCount
+    @TestCaseMetadata(
+        description="""
+        This test case stops VM resizes the node, starts VM and checks if it has
+        the expected capabilities (memory size and core count) after the resize
+
+        Steps:
+        1. Resize vm into larger vm size
```
Comment (LiliDeng) [inline microsoft/testsuites/core/vm_resize.py]: add stop and start vm into steps

Comment (adityagesh) [reply microsoft/testsuites/core/vm_resize.py]: added

File: microsoft/testsuites/core/vm_resize.py [hunk 2]
```diff
@@ -59,12 +59,58 @@ def verify_vm_hot_resize(self, log: Logger, node: Node) -> None:
...
+    @TestCaseMetadata(
+        description="""
+        This test case stops VM resizes the node, starts VM and checks if it has
+        the expected capabilities (memory size and core count) after the resize
+
+        Steps:
+        1. Stop VM
+        2. Resize vm into larger vm size
+        3. Start VM
+        4.. Check the node's core count and memory size against their expected values
```
Comment (LiliDeng) [inline microsoft/testsuites/core/vm_resize.py]: one extra dot

File: microsoft/testsuites/core/vm_resize.py [hunk 3]
```diff
@@ -59,12 +59,58 @@ def verify_vm_hot_resize(self, log: Logger, node: Node) -> None:
...
     def verify_vm_hot_resize_decrease(self, log: Logger, node: Node) -> None:
-        self._verify_vm_hot_resize(node, ResizeAction.DecreaseCoreCount)
+        self._verify_vm_resize(node, ResizeAction.DecreaseCoreCount)
 
-    def _verify_vm_hot_resize(
-        self, node: Node, resize_action: ResizeAction = ResizeAction.IncreaseCoreCount
+    @TestCaseMetadata(
+        description="""
+        This test case stops VM resizes the node, starts VM and checks if it has
+        the expected capabilities (memory size and core count) after the resize
```
Comment (LiliDeng) [inline microsoft/testsuites/core/vm_resize.py:69]: This test case stops the VM, resizes the VM, starts the VM, then ...

File: microsoft/testsuites/core/vm_resize.py [hunk 4]
```diff
@@ -76,7 +124,7 @@ def _verify_vm_hot_resize(
                 if "no available size for resizing" in str(identifier):
                     raise SkippedException(str(identifier))
                 if (
-                    "cannot find current vm size in eligible list" in str(identifier)
+                    "cannot find current VM size in eligible list" in str(identifier)
```
Comment (LiliDeng) [inline microsoft/testsuites/core/vm_resize.py]: don't change this

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**